packages/okta-vue/README.md

```diff
@@ -0,0 +1,142 @@
+# Okta Vue SDK
+The Okta Vue SDK is a wrapper around [Okta Auth JS](https://github.com/okta/okta-auth-js), that builds on top of Okta's [OpenID Connect API](https://developer.okta.com/docs/api/resources/oidc.html).
+
+This library currently supports:
+  - [OAuth 2.0 Implicit Flow](https://tools.ietf.org/html/rfc6749#section-1.3.2)
+
+## Getting Started
+* If you do not already have a **Developer Edition Account**, you can create one at [https://developer.okta.com/signup/](https://developer.okta.com/signup/).
+* If you don't have a Vue app, or are new to Vue, please start with the [Vue Quickstart](https://vuejs.org/v2/guide/#Getting-Started) guide. It will walk you through the creation of a Vue app, creating routes, and other application development essentials.
```

Comment:
This Vue quickstart is a naive html+js app, aka not a webpack app that you can easily add node modules to.  Is there a better place to point folks, so that they're read to install our lib as a node module?